doc/varnish-configuration.rst

```diff
@@ -40,14 +40,19 @@ To configure Varnish for `handling PURGE requests <https://www.varnish-cache.org
 
 Purge removes a specific URL (including query strings) in all its variants (as specified by the ``Vary`` header).
 
+.. code-block::
+    include "path-to-config/varnish-version/fos_purge.vcl"
+
 .. configuration-block::
 
-    .. literalinclude:: ../tests/Functional/Fixtures/varnish-4/purge.vcl
+    .. literalinclude:: ../tests/Functional/Fixtures/varnish-4/fos.vcl
         :language: varnish4
+        :lines: 17,19,21-22
         :linenos:
 
-    .. literalinclude:: ../tests/Functional/Fixtures/varnish-3/purge.vcl
+    .. literalinclude:: ../tests/Functional/Fixtures/varnish-3/fos.vcl
         :language: varnish3
+        :lines: 17,19-20,25-32
```

Comment:
Yeah, was wondering the same. While I’m all in favour of including (tested) VCL in our docs, just those couple of lines may not be worth the effort.